packages/ra-ui-materialui/src/form/SimpleFormIterator.js

```diff
@@ -115,6 +115,7 @@ export class SimpleFormIterator extends Component {
             translate,
             disableAdd,
             disableRemove,
+            TransitionProps,
```

Comment:
Can you rename it to `transitionProps`?

Reply:
I used material-ui naming pattern here - see `ExpansionPanel` `TransitionProps` - https://material-ui.com/api/expansion-panel/#props